controllers/storagecluster/cephobjectstores.go

```diff
@@ -19,6 +19,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const EnableRGWAutoscaleAnnotation = "ocs.openshift.io/enable-RGW-autoscale"
```

Comment:
```suggestion
const EnableRGWAutoscaleAnnotation = "ocs.openshift.io/enable-rgw-autoscale"
```
I don't see the need to capitalize as hyphens are already being used for separation (and removes the need to change case while typing as well 😅)